cranelift/filetests/filetests/isa/riscv64/wasm/zbb.wat

```diff
@@ -0,0 +1,88 @@
+;;! target = "riscv64"
+;;! compile = true
+;;! settings = ["has_zbb", "opt_level=speed"]
```

Comment:
I think we can also write a wasm -> clif version of this test by not using `compile = true`. That would be kinda nice since the rule itself is target independent.